docker-compose.solr.yaml

```diff
@@ -23,10 +23,24 @@ services:
     volumes:
       - .:/mnt/ddev_config
       - solr:/var/solr
-    command: bash -c "docker-entrypoint.sh solr zk cp file:/mnt/ddev_config/solr/security.json zk:/security.json && exec solr-foreground"
+    command: bash -c "
+      docker-entrypoint.sh solr zk cp file:/mnt/ddev_config/solr/security.json zk:/security.json;
+      export SOLR_AUTH_TYPE='basic';
+      export SOLR_AUTHENTICATION_OPTS='-Dbasicauth=solr:SolrRocks';
```

Comment:
It seems like these would be easier to add in the docker-compose.solr.yaml

Reply:
I don't want to set them in the container. `commands/solr/solr` should not use these credentials!

Reply:
OK, I reviewed the authentication stuff. Maybe it is more convenient to keep solr:SolrRocks everywhere.